src/ParametersHandler/include/BipedalLocomotionControllers/ParametersHandler/IParametersHandler.h

```diff
@@ -10,56 +10,204 @@
 
 #include <memory>
 #include <string>
+#include <vector>
+
+#include <BipedalLocomotionControllers/GenericContainer/Vector.h>
+#include <BipedalLocomotionControllers/GenericContainer/TemplateHelpers.h>
 
 namespace BipedalLocomotionControllers
 {
 namespace ParametersHandler
 {
 
+
+template <typename T>
+struct is_string : public std::disjunction<std::is_same<char*, typename std::decay<T>::type>,
+                                           std::is_same<const char*, typename std::decay<T>::type>,
+                                           std::is_same<std::string, typename std::decay<T>::type>>
+{
+};
```

Comment:
What about moving this to the ``TemplateHelpers`` file since it is pretty generic?

Reply:
Done in bf7e67e5b0145e76983c4e3d6edf83e0c544646f